code-studio/build-js.js

```diff
@@ -48,7 +54,22 @@ build_commands.execute([
     shouldFactor: false,
     shouldMinify: commander.min,
     shouldWatch: commander.watch
+  }),
+
+  // Build embedVideo.js in its own step (skipping factor-bundle) so that
+  // we don't have to include the large code-studio-common file in the
+  // embedded video page, keeping it fairly lightweight.
+  // (I wonder how much more we could slim it down by removing jQuery!)
+  build_commands.browserify({
+    srcPath: SRC_PATH,
+    buildPath: BUILD_PATH,
+    filenames: [
+      'embedVideo.js'
+    ],
+    commonFile: 'embedVideo',
+    shouldFactor: false,
+    shouldMinify: commander.min,
+    shouldWatch: commander.watch
```

Comment:
my copy-paste-bloat senses are starting to tingle a bit here with this third `build_commands.browserify` entry. Maybe in a future pass we can collapse this set of verbose javascript configuration lines into a more compact declarative format before they spiral too out of control?